scripts/admincommands.js

```diff
@@ -426,32 +425,30 @@ exports.handleCommand = function(src, command, commandData, tar, channel) {
     }
     // hack, for allowing some subset of the owner commands for super admins
     if (isSuperAdmin(src)) {
-       if (["changeauth"].indexOf(command) != -1) {
-           normalbot.sendMessage(src, "Can't aboos some commands", channel);
-           return;
-       }
-       return require("ownercommands.js").handleCommand(src, command, commandData, tar, channel);
+        if (["changeauth"].contains(command)) {
+            normalbot.sendMessage(src, "Can't aboos some commands.", channel);
+            return;
+        }
+        return require("ownercommands.js").handleCommand(src, command, commandData, tar, channel);
     }
 
     return "no command";
 };
-exports.help = 
-    [
-        "/ban: Bans a user.",
-        "/unban: Unbans a user.",
-        "/toggleweblinks [on/off]: Allows or disallows webclient users to send clickable urls.",
-        "/memorydump: Shows the state of the memory.",
-        "/nameban: Adds a regexp ban on usernames.",
-        "/nameunban: Removes a regexp ban on usernames.",
-        "/channelnameban: Adds a regexp ban on channel names.",
-        "/channelnameunban: Removes a regexp ban on channel names.",
-        "/namewarn: Adds a regexp namewarning",
-        "/nameunwarn: Removes a regexp namewarning",
-        "/destroychan: Destroy a channel (official channels are protected).",
-        "/indigoinvite: To invite somebody to staff channels.",
-        "/indigodeinvite: To deinvite unwanted visitors from staff channel.",
-        "/cookieban: Bans an online target by cookie.",
-        "/cookiemute: Puts an online android target on an autosmute list by cookie.",
-        "/cookieunban/mute: Undos a cookieban/mute. Will take effect when they next log in",
-        "/watchlog: Search the watch log. Accepts /watch 15 (last 15 entries), /watch 10-20 (last 10 to 20) and /watch 10:[Word] (entries that contain that word)."
-    ];
+exports.help = [
+    "/ban: Bans a user.",
+    "/unban: Unbans a user.",

```

Comment:
Combine with previous line with like:

/ban: Bans a user. Use /unban to unban.